Exporter/src/main/java/uk/ac/ebi/biostd/exporter/service/SubmissionService.java

```diff
@@ -106,13 +106,24 @@ private Boolean hasSameValue(Attribute attribute, String value) {
 
     private Section processSection(Section section) {
         long sectionId = section.getId();
-        section.setAttributes(sectionDao.getSectionAttributes(section.getId()));
+        section.setAttributes(getSectionAttributes(section));
         section.setFiles(getSectionFiles(sectionId));
         section.setLinks(getSectionLinks(sectionId));
         section.setSubsections(getSubsections(sectionId));
         return section;
     }
 
+    private List<Attribute> getSectionAttributes(Section section) {
+        String fileListName = section.getFileListName();
+        List<Attribute> attributes = sectionDao.getSectionAttributes(section.getId());
+
+        if (fileListName != null && !fileListName.isEmpty()) {
```

Comment:
I will use Strings.isNotEmpty. Also I think you need to filterout attribute if it exists so it does not appear duplicated